device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/ClientOptions.java

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.azure.sdk.iot.device;
+
+import lombok.Getter;
+import lombok.Setter;
+
+public final class ClientOptions
+{
+    /**
+     * The Plug and Play Model Id associated with the device identity.
```

Comment:
Add a note that non plug and play users should not set this value

Reply:
will do